apigw-lambda-translate/template.yaml

```diff
@@ -0,0 +1,36 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: An Amazon REST API gateway integrated with a AWS Lambda function that calls AWS Translate service to translate given text into specified language.
+Resources:
+  # API Gateway REST API
+  ApiGatewayApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      Name: ApiGatewayApi
+      StageName: prod
+      EndpointConfiguration:
+        Type: REGIONAL
+  # Lambda Function to call AWS Translate service
+  ApiFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: apigw-translate
+      Runtime: python3.9
+      Handler: index.lambda_handler
+      CodeUri: src/
+      MemorySize: 128
+      Policies:
+        - TranslateReadOnly
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Path: /
+            Method: post
+            RestApiId:
+              Ref: ApiGatewayApi
+
+    Outputs:
```

Comment:
not properly indented and this doesnt shows the output. please fix

Reply:
fixed indentation and outputs. Thanks